core/domain/html_validation_service_test.py

```diff
@@ -1248,3 +1248,10 @@ def test_no_convertion_of_non_interactive_image_tags(self):
                 html_validation_service.convert_svg_diagram_tags_to_image_tags(
                     test_case['html_content']),
                 test_case['expected_output'])
+
+    def test_get_html_string_with_incorrectly_encoded_chars_correct_html_string(self): # pylint: disable=line-too-long
```

Comment:
Try to find a name that isn't too long, but still communicates the intention of the test.

Reply:
Thanks @seanlip. Tried `test_fix_incorrectly_encoded_html_string( )`. I think it reflects the purpose of the method it is testing.